Sources/Sentry/include/SentryTracer+Private.h

```diff
@@ -5,4 +5,10 @@ SentryTracer ()
 
 @property (nonatomic, strong) SentryHub *hub;
 
+/**
+ * We need an unchanging identifier to track concurrent tracers against a static profiler.
+ * @c SentryTracer.traceId can be changed by consumers so is unfit for this purpose.
+ */
+@property (nonatomic, strong) SentryId *concurrencyID;
```

Comment:
`l`: I think we could use the ID also for other purposes than concurrency. We could also call it something like `immutableID`, `immutableTraceID`, etc.

Reply:
i'll call it `internalID`